juno.mk

```diff
@@ -0,0 +1,192 @@
+DEBUG ?= 0
+
+-include common.mk
+
+################################################################################
+# Mandatory definition to use common.mk
+################################################################################
+CROSS_COMPILE_NS_USER	?= "$(CCACHE)$(AARCH64_CROSS_COMPILE)"
+CROSS_COMPILE_NS_KERNEL	?= "$(CCACHE)$(AARCH64_CROSS_COMPILE)"
```

Comment:
It's GitHub tricking Pascal here, it looks like there is a space (tab) between the `"` and `$(CCACHE)"`, but in fact there aren't if you're look at the raw patch.